app/forms/sufia/forms/work_form.rb

```diff
@@ -4,6 +4,12 @@ class WorkForm < CurationConcerns::Forms::WorkForm
 
     self.terms += [:collection_ids]
 
+    def initialize(model, current_ability)
+      # Default agreement to accepted if users have already uploaded files to the work
+      @agreement_accepted = model.file_sets.count > 0
```

Comment:
I assume you need an accessor for `agreement_accepted`. This seems like it might be a very expensive way (lots of fedora hits) to determine if the agreement is accepted. Might we just assume that if model is persisted the agreement was accepted?